WireAuthentication/Sources/WireAuthentication/Components/VerificationEmailCodeComponent.swift

```diff
@@ -37,16 +39,25 @@ final class VerificationEmailCodeComponent: Component<VerificationEmailCodeCompo
     private let email: String
     private let password: String
     private let name: String
+    private let isDataUsageAgreementAccepted: Bool
+    private let registrationAnalyticsTracker: (any RegistrationAnalyticsTrackerProtocol)?
+    private let registrationAnalyticsIDRepository: any RegistrationAnalyticsIDRepositoryProtocol
 
     init(
         parent: any Scope,
         email: String,
         password: String,
-        name: String
+        name: String,
+        isDataUsageAgreementAccepted: Bool,
+        analyticsEventTracker: (any RegistrationAnalyticsTrackerProtocol)?,
+        analyticsIDRepository: any RegistrationAnalyticsIDRepositoryProtocol
```

Comment:
issue: these don't need to be injected as you have already declared them as needle dependencies for the component, so you can access them via `dependency.analyticsEventTracker` and `depedency.alayticsIDRepository`.